### DISCUSSION TITLE
DAX Studio Configuration

### DISCUSSION THREAD

**Discussion by Shantanu3006** [DaxStudio/DaxStudio#1043] [Q&A] [answered]
Hi 
I am curious to know how and where the configuration settings (like Privacy settings and other preferences)  of DAX studio are saved on a windows machine. Can someone please help with this info?

Thanks
Shantanu

**Comment by dgosbell** [comment, marked as answer]
If you install DAX Studio the config settings are stored in the registry under HKEY_CURRENT_USER\SOFTWARE\DaxStudio

The only exception to this is if you do an "All Users" install (requiring admin rights) - and the setting to block all internet access is chosen, then this is written to HKEY_LOCAL_MACHINE\SOFTWARE\DaxStudio (which requires admin rights to change)

If you use the portal version then a settings.json file is created in the same folder as daxstudio.exe and any non-default settings are stored there.

**Reply by Shantanu3006** [reply]
Thank you Darren ! This information is so much helpful.

**Reply by marcosqlbi** [reply]
@dgosbell I created a pull request to add this info to the security-privacy page in the documentation

**Reply by dgosbell** [reply]
Great idea! I've just merged that pull request it should be live in a few minutes

**Reply by Shantanu3006** [reply]
Thanks Darren and Marco ! I see this information is now available on the DAX Studio website.
